stack.yaml

```diff
@@ -2,4 +2,5 @@ packages:
 - '.'
 - sample-cabal-project/
 extra-deps: []
-resolver: lts-4.1
+# resolver: lts-4.1
+resolver: nightly-2016-06-15
```

Comment:
Since the cairo example does not work with nightly, I wouldn't have it as the default one. Feel free to bump it to lts 6 though.